src/main/java/model/data/process/functions/ValueFunction.java

```diff
@@ -20,4 +30,26 @@ public ValueFunction(DataTable model, DataDescriber<NumberValue> argument) {
 	 */
 	public abstract DataValue calculate();
 	
+	/**
+	 * initialize class checks if specified column is eligible
+	 * @return DataValue because it calls calculate
+	 */
+	public void initialize() {
+		if(table.getRowCount() == 0)
```

Comment:
Het is mooier om in dit soort gevallen, alsnog de brackets te zetten, omdat mensen die het aanpassen misschien niet goed op letten.